src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-inline: 16px;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
```

Comment:
bg image is not suitable here since this pic has semantic meaning
we can use bg image for some decorative elements